app/src/main/java/org/oppia/android/app/splash/SplashActivity.kt

```diff
@@ -28,6 +39,7 @@ class SplashActivity :
   AppCompatActivity(),
   FragmentComponentFactory,
   DeprecationNoticeExitAppListener,
+  DeprecationNoticeActionListener,
```

Comment:
Perhaps this should come in a later PR when the SplashActivity can actually shows these dialogs (otherwise it's a bit tough to make sure this functionality gets tested later on when things are fully hooked up).

Reply:
I have removed this as well as its implementation from the changes. I will add it back in the next milestone.